charts/opssight/templates/secrets.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+data:
+  blackduck.json: {{ .Values.secret.externalHub }}
```

Comment:
add b64enc method like below. check for alert or blackduck helm chart. input should be not encoded
blackduck.json: {{ .Values.secret.externalHub | b64enc }}

Reply:
same changes to all secret values